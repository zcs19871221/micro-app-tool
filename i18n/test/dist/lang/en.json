{
  "key0001": "默认名称",
  "key0002": "男",
  "key0003": "好的",
  "key0004": "不好",
  "key0005": "你好",
  "key0006": "哈哈",
  "key0007": "为什么",
  "key0008": "北京大学",
  "key0009": "真是男的",
  "key0010": "北京",
  "key0011": "大学",
  "key0012": "套娃",
  "key0013": "套娃3",
  "key0014": "真不错",
  "key0015": "租户不存在",
  "key0016": "请重新登录，或联系系统管理员创建至少一个租户",
  "key0017": "知道了",
  "key0018": "别名",
  "key0019": "别名2",
  "key0020": "意义咋样",
  "key0021": "请输入",
  "key0022": "中纯文本",
  "key0023": "然后分割"
}