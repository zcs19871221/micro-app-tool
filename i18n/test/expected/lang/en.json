{
  "key0001": "默认名称",
  "key0002": "男",
  "key0003": "好的",
  "key0004": "不好",
  "key0005": "你好",
  "key0006": "哈哈",
  "key0007": "为什么",
  "key0008": "北京大学",
  "key0009": "真是男的",
  "key0010": "北京",
  "key0011": "大学",
  "key0012": "套娃",
  "key0013": "套娃3",
  "key0014": "真不错",
  "key0015": "租户不存在",
  "key0016": "请重新登录，或联系系统管理员创建至少一个租户",
  "key0017": "知道了",
  "key0018": "和",
  "key0019": "别名",
  "key0020": "别名2",
  "key0021": "意义咋样",
  "key0022": "请输入",
  "key0023": "中纯文本",
  "key0024": "然后分割",
  "key0025": "空节点"
}